Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -308,6 +308,10 @@ define([
 
         if (gltfFormat === 0) {
             var gltfUrl = getStringFromTypedArray(gltfView);
+
+            // We need to remove padding from the end of the model URL in case this tile was part of a composite tile
+            // This removes all white space and null characters from the end of the string.
+            gltfUrl = gltfUrl.trim().replace(/\0+$/, '');
```

Comment:
Because `trim` and `\s` only match whitespace characters which according to the docs are:
```
A space character
A tab character
A carriage return character
A new line character
A vertical tab character
A form feed character
```
`null` would be a common pad character as well as a lot of languages initialize buffers to 0.